lms/djangoapps/verify_student/models.py

```diff
@@ -637,6 +644,14 @@ class SoftwareSecurePhotoVerification(PhotoVerification):
     # encode that. The result is saved here. Actual expected length is 344.
     photo_id_key = models.TextField(max_length=1024)
 
+    # This is a model field only used for testing, and is used to mock saving and
+    # reading from S3.
+    encoded_face_data = models.BinaryField(null=True)
```

Comment:
The reason why I added these fields and the settings variable was to use for both local testing and unit testing. Without these I don't think it would have been possible for me to replicate the endpoint locally outside of unit testing. If this becomes a support tool on stage with a UI at some point, I think it's useful to be able to use the process in devstack. 

If you don't think that's necessary I can definitely change to just mocking the functions without saving the data

Reply:
I think we can have another way to tackle the devstack testing problem than putting the encrypted data into database. I think we can solve that problem later. 
For now, try to make sure unit testing works with mock on function, then I think we can execute the full experience testing on stage. 
Later on, let's figure out why on devstack, we cannot just use MEDIA_ROOT and local storage, see https://github.com/edx/edx-platform/blob/master/lms/envs/devstack.py#L21

Reply:
Makes sense, I'll rework this so that it's only mocking